companies/crows-crows-crows.json

```diff
@@ -0,0 +1,22 @@
+{
+    "slug": "crows-crows-crows",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Roberts & Co Bunsen Outlet Limited",
+    "runs": [
+        "Crows Crows Crows"
+    ],
+    "address": "54 Middle Street\nSowerby Bridge\nWest Yorkshire\nHX6 3NX\nUnited Kingdom",
+    "email": "hello@crowscrowscrows.com",
+    "web": "https://crowscrowscrows.com",
+    "sources": [
+        "https://crowscrowscrows.com/legal.html",
+        "https://crowscrowscrows.com/privacy/"
+    ],
+    "suggested-transport-medium": "email",
```

Comment:
If the email address is not privacy-specific, we don't set the stm to `email`, even if the email is explicitly mentioned in the privacy policy.